src/components/PrimaryNav.tsx

```diff
@@ -123,15 +124,29 @@ const PrimaryNav = (): JSX.Element => {
         )}
 
         {!user && (
-          <button
-            className="bg-osOrange w-[64px] h-[24px]  rounded-[6px] px-[12px] py-[2px] text-xs font-semibold text-white"
-            onClick={async () => {
-              capturePostHogAnayltics("User Login", "userLoginAttempt", "true");
-              await signIn({ provider: "github" });
-            }}
-          >
-            Sign in
-          </button>
+          <div className="flex justify-around">
+            <div className="hidden md:flex items-center text-osGrey">
+              <a
+                href="https://github.com/open-sauced/hot"
+                rel="noreferrer"
+                target="_blank"
+              >
+                <AiOutlineStar className="justify-self-start inline-block mr-[10px]" />
+
+                <span className="text-sm">Star us on GitHub</span>
+              </a>
+            </div>
+
+            <button
+              className="bg-osOrange w-[64px] h-[24px]  rounded-[6px] px-[12px] py-[2px] text-xs font-semibold text-white md:ml-[20px]"
```

Comment:
This pattern of hard coding px is an anti-pattern of tailwind. https://github.com/open-sauced/hot/issues/267